### PR TITLE
Save Resume - Completing the Server side

### PR DESCRIPTION
**Pull request by guyshany**
Completed the server side of saving a new user resume (fixed postgres queries), and added some client logic

### CODE REVIEW

File: scripts/DDL.sql [hunk 1]
```diff
@@ -39,13 +39,24 @@ CREATE TABLE jobs.base_locations(
 CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
 	display_name text,
-	last_entrance timestamp,
+	last_entrance timestamp
+);
+
+CREATE TABLE jobs.users_resume(
+	upn text PRIMARY KEY,
```
Comment (michael-hoffman-26) [inline scripts/DDL.sql:46]: why not FK to Users?

File: server/routes/api/user.js [hunk 2]
```diff
@@ -7,27 +7,31 @@ const router = express.Router();
 router.get('/', async (req, res) => {
     const { upn } = req.params;
 
-    const { rows } = await db.query(`
+    const { rows } = await db.query(
+        `
```
Comment (ShaharAnati) [inline server/routes/api/user.js:11]: If you break line for opening `, please break line for the closing one (on line 17) as well, the same way you did on lines 72-75
repeats itself on lines: 28-32, 49-51, 136-138, 148-169

**Comment by ShaharAnati** [comment]
Big MR, was it a WIP before that I missed?

**Comment by guyshany** [comment]
No, never saw the point of WIP MR's

File: scripts/DDL.sql (hunk 1 above)
Comment (guyshany) [reply scripts/DDL.sql:46]: Thanks

File: server/routes/api/user.js (hunk 2 above)
Comment (guyshany) [reply server/routes/api/user.js:11]: :+1:

**Pull request merged**